DayrSuryan/DSEthiop7.xml

```diff
@@ -349,10 +412,19 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                      The manuscript could be copied
                      by <persName ref="PRS11959Pawlos" role="scribe"/>,
                      in <placeName ref="LOC2804Egypt">Egypt</placeName> or in <placeName ref="LOC2991Eritre">Eritrea</placeName>.
-                     A brief note identifies <persName ref="PRS12557Barha">ḫalaqā Barha</persName> as the owner.
+                     A brief note identifies <persName ref="PRS12557Barha">ḫalaqā Barha</persName>, who is also mentioned in
+                     <ref target="#coloph1">a colophon</ref>as the owner.
                    </origin>
-                  <provenance/>
+                 
                </history>
+               
+               <additional>
+                  <adminInfo>
+                     <note>A paper sheet containing a modern description of the manuscript in Arabic is placed inside it.
+                        It is stated in the description this manuscript was copied from <persName ref="PRS12557Barha"/> as a
```

Comment:
copied from a book? copied by someone?

Reply:
It is not clear to me in the Arabic text, this as literal a paraphrase as I managed

Reply:
Then you indicate that, smth like: "A recent paper sheet containing a brief (not quite clear) description of the manuscript in Arabic is placed... It states that the manuscript was copied..."